### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DmACKGL**
Added:
    -issues link
    -Generic CS 1.6 Logo [Change]
    -Build Status
    -Android specification ([Android])

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,16 @@
-# cs16-client
+![CS1.6 Logo](https://media.toggle.com/000/970/956/a26_6c4_60_60-counter-strike.jpg)
```
Comment (a1batross) [inline README.md]: Do you think that this logo doesn't have any copyright problems? I've recommended a non-copyrighted icon of cs16-client. This is here: https://github.com/SDLash3D/cs16-client/blob/master/android/res/drawable-xhdpi/ic_launcher.png

Drawn by @sergiopoverony.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
 * Based on HLSDK 2.3. 
 * Works without VGUI.
 * Runs on your phone.
 * Helpful for modders.
 
-Write your issues here. 
+
+
+##### ¿Any Problem, question or suggestion?
+ Write your issues [here](https://github.com/SDLash3D/cs16-client/issues)
```
Comment (a1batross) [inline README.md]: Issues is closed now due to spam.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
+
+###### Build Status: ¡Beta!
```
Comment (a1batross) [inline README.md]: Lol, static build status string. We're out of beta, also.

Comment (DmACKGL) [inline README.md]: "Counter-Strike 1.6 rewritten client.dll. Without VGUI, ParticleMan. And ecology friendy. BETA TESTING NOW."
I use this information

Comment (a1batross) [inline README.md]: I forgot about it. Fixed.

**Comment by a1batross** [comment]
I updated it by myself.

**Pull request closed without merging**